src/directives/layertree.js

```diff
@@ -4,15 +4,25 @@ goog.provide('ngeo.layertreeDirective');
 goog.require('ngeo');
 
 
+/**
+ * Unique template path for the layertree;
+ * @type {string|undefined}
+ */
+ngeo.layertreeTemplateUrlValue = undefined;
```

Comment:
I've retried, and it seems that I definitly must use this value :-/

gmf layertree template makes **ngeo** trees (each node is one):
 - If I set a ngeo-layertree-template, the path doesn't work in hosted-examples
 - If I don't set one, the default template will be the default **ngeo** template.

Layertrees (except root that is defined in the gmf layertree directive) can't use a gmf layertree template. I must keep the first given path.